youtube.module

```diff
@@ -588,7 +588,7 @@ function youtube_field_widget_form(&$form, &$form_state, $field, $instance, $lan
 
   // Add our own description if one is not provided by the UI.
   if ($element['#description'] == '') {
-    $element['input']['#description'] = t('Valid URL formats include: https://www.youtube.com/watch?v=1SqBdS0XkV4&t=120, https://youtube.com/live/1SqBdS0XkV4 and https://youtu.be/1SqBdS0XkV4');
+    $element['input']['#description'] = t('Valid URL formats include: https://www.youtube.com/watch?v=1SqBdS0XkV4&t=120, https://youtube.com/live/1SqBdS0XkV4 and https://youtu.be/1SqBdS0XkV4&t=1h30m20s');
```

Comment:
Since the description text is changed anyway and the long line is hard to read, I suggest to format it as list. I've tested this with browser dev tools, and it looks much better, in my opinion. See screenshot below. Do you prefer a separate issue for this suggestion?

<img width="548" alt="youtube-field-valid-urls-list" src="https://github.com/backdrop-contrib/youtube/assets/14188307/9ded8063-ed4e-46bf-96b1-afbd80b506f3">

Reply:
Good suggestion. Thanks